grails-app/views/stockMovement/list.gsp

```diff
@@ -53,13 +53,13 @@
 
         <div class="right">
             <div class="button-container">
-                <g:if test="${params.direction as StockMovementType == StockMovementType.INBOUND}">
+                <g:if test="${params.direction as org.pih.warehouse.api.StockMovementDirection == StockMovementDirection.INBOUND}">
```

Comment:
I know this isn't on you @pmuchowski but this definitely a place where we probably could / should have create a separate GSPs for the different stock movement types / directions.